src/maintainer/knowledge_base.rst

```diff
@@ -210,25 +210,14 @@ In conda_build_config.yaml file:
 
 .. code-block:: yaml
 
-    c_compiler:
-    - vs2019
-    cxx_compiler:
-    - vs2019
+c_compiler:  # [win]
+- vs2019  # [win]
+cxx_compiler:  # [win]
+- vs2019  # [win]
```

Comment:
```suggestion
    c_compiler:    # [win]
    - vs2019       # [win]
    cxx_compiler:  # [win]
    - vs2019       # [win]
```

I think you need the initial indentation for it to render correctly in `.rst` files.

Reply:
Thanks, fixed!